pages/linux/apt.md

```diff
@@ -0,0 +1,31 @@
+# apt
+
+> Package management utility for Debian based distributions. 
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent apt commands:
```

Comment:
"first, " can be removed to shorten the sentence. Also "synchronize" --> "update" and "subsequent" --> "further" (mostly to shorten the sentence some more).